src/yunohost/app.py

```diff
@@ -161,79 +156,99 @@ def app_list(offset=None, limit=None, filter=None, raw=False, installed=False, w
         app_fetchlist()
         applists = app_listlists()['lists']
 
+    # Construct a dictionnary of apps, based on known app lists
     for applist in applists:
         with open(os.path.join(repo_path, applist + '.json')) as json_list:
-            for app, info in json.loads(str(json_list.read())).items():
-                if app not in app_dict:
-                    info['repository'] = applist
-                    app_dict[app] = info
+            for app, info in json.load(json_list).items():
+                if app in app_dict:
+                    continue
 
+                info['repository'] = applist
+                app_dict[app] = info
+
+    # Get app list from the app settings directory
     for app in os.listdir(apps_setting_path):
         if app not in app_dict:
-            # Look for forks
+            # Handle multi-instance case like wordpress__2
             if '__' in app:
                 original_app = app[:app.index('__')]
                 if original_app in app_dict:
                     app_dict[app] = app_dict[original_app]
                     continue
-            with open( apps_setting_path + app +'/manifest.json') as json_manifest:
-                app_dict[app] = {"manifest":json.loads(str(json_manifest.read()))}
+                # FIXME : What if it's not !?!?
```

Comment:
This FIXME isn't needed. Because the goal of this if is to fill app_dict[app] if it is not already filled. But it is the case with the first setup of 'wordpress' (or other apps)